pkgs/games/atlauncher/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchurl, jre, lib, makeWrapper, stdenv }:
+
+let
+  version = "3.4.28.1";
```

Comment:
As you are using a recursive attrset (that rec thing) you dont need to put version in a let expression before.